presqt/api_v1/serializers/target.py

```diff
@@ -43,3 +44,20 @@ class TargetSerializer(serializers.Serializer):
     name = serializers.CharField(max_length=256)
     supported_actions = SupportedActions()
     supported_hash_algorithms = serializers.StringRelatedField(many=True)
+    detail = serializers.SerializerMethodField()
+
+    def get_detail(self, instance):
```

Comment:
I'd like to discuss an idea about group embedded URLs into a `links` array.
There is a UI reason for this.